hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/ZKReplicationPeerStorage.java

```diff
@@ -185,6 +185,17 @@ public ReplicationPeerConfig getPeerConfig(String peerId) throws ReplicationExce
     }
   }
 
+  @Override
+  public long getPeerCreateTime(String peerId) throws ReplicationException {
+    long createTimeIfNodeExists;
+    try {
+      createTimeIfNodeExists = ZKUtil.getCreateTimeIfNodeExists(zookeeper, getPeerNode(peerId));
```

Comment:
Why not just return here? And what is the return value if the znode does not exist?